ui/app/AppLayouts/Chat/ChatLayout.qml

```diff
@@ -321,9 +322,21 @@ StackLayout {
             Connections {
                 target: root.rootStore.communitiesModuleInst
 
-                function onSharedAddressesForAllNonKeycardKeypairsSigned() {
+                function onAllSharedAddressesSigned() {
```

Comment:
Just for me to better understand the complete flow.. Could you describe which are the scenarios that make this signal to be triggered?

So we can have 1 unique autentication method OR multiple ones (like by password + by keycard). In case of multiple ones, this process is done completely decoupled and async (meaning that the user can order the authentication for each type in a different time) so, WHEN is it considered that `all shared addresses have been signed` in this "multiple authentication methods" scenario?

Reply:
It will be emitted when all selected addresses are signed, regardless of the number of selected addresses that user wants to reveal.